index.html

```diff
@@ -35,12 +35,235 @@ <h2 class="headline-title2">I'm a Software Developer!</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">BOOTSTRAP</a>
+                    </li>
+                    <li>
+                        <a href="#">RUBY</a>
+                    </li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+
+        <section class="about">
+            <article class="about-content">
+                <div class="about-header">
+                    <h2>About Me</h2>
+                    <p>
+                        Hello I’m a software developer! I can help you build a product,
+                        feature or website Look through some of my work and experience! If
+                        you like what you see and have a project you need coded, don’t
+                        hesitate to contact me.
+                    </p>
+                    <button type="button">Get my resume</button>
+                </div>
+                <div class="about-footer">
+                    <hr />
+                    <ul class="major-menu">
+                        <li>
+                            <div class="lang-first">
+                                <img src="images/rhombus.png" width="18px" height="18px" alt="" />
+                                <h2>Language</h2>
+                            </div>
+                            <ul class="minor-menu">
+                                <li class="jss">Javascript</li>
+                                <li class="jss">Ruby</li>
+                                <li class="jss">Html</li>
+                                <li class="jss">Css</li>
+                            </ul>
+                        </li>
+                        <li>
+                            <div class="lang-sec">
+                                <img src="icons/rectangle.png" width="18px" height="18px" alt="" />
+                                <h2>Frameworks</h2>
+                            </div>
+                            <ul class="minor-menu">
+                                <li class="jss">Bootstrap</li>
+                                <li class="jss">Ruby on Rails</li>
+                                <li class="jss">RSPec</li>
+                                <li class="jss">CapyBara</li>
+                                <li class="jss">Selenium</li>
+                            </ul>
+                        </li>
+                        <li>
+                            <div class="lang-sec">
+                                <img src="icons/circle.png" width="18px" height="18px" alt="" />
+                                <h2>Skills</h2>
```

Comment:
- [ ] As I mentioned above :point_up_2: please stick to the Figma design there are some spacing issues so kindly fix these issues :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

| Your Design | Figma Design |
|--------|--------|
| ![screencapture-127-0-0-1-5500-2023-04-30-01_38_13 (1)](https://user-images.githubusercontent.com/45226848/235324265-f61b8699-0524-4937-be5f-f5266130f550.png) | ![Screenshot from 2023-04-30 01-45-10](https://user-images.githubusercontent.com/45226848/235324267-642a923a-6e9b-49d5-bc44-bf2af297726a.png) | 

- [ ] If you check your `Languages`, `Frameworks` and `Skills` text is very big so kindly reduce the text size :+1: 
- [ ] Also, if you check `Frameworks` and `Skills` there are icons missing so kindly fix this issue as well :+1: